### DISCUSSION TITLE
Stdlib documentation

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/stdlib#599] [General]
Now that the new documentation for fpm is up at https://fpm.fortran-lang.org/, what do you think about adopting a similar framework for stdlib as well?

**Comment by ivan-pi** [comment]
Unfortunately I missed the last Fortran meeting. Generally speaking, the look and feel of the fpm documentation is better than the current FORD output. I'd be in favor of adopting a similar framework. 

Speaking of documentation, the new matplotlib documentation looks excellent: https://matplotlib.org/

**Reply by awvwgk** [reply]
Nice, this looks like based on pydata-sphinx-theme, but heavily modded with CSS (actually exactly the kind of modifications I'd need for the fortran-lang.org webpage, definitely going to steal this one).

**Reply by awvwgk** [reply]
They have actually their own sphinx theme (https://github.com/matplotlib/mpl-sphinx-theme) to customize the pydata one

**Reply by milancurcic** [reply]
I agree. FWIW, the original fortran-lang.org site was a direct adaptation of https://neovim.io/. We could take the same approach now. I'd be happy to help port much of the content to the new framework if we choose to do so.

**Comment by awvwgk** [comment]
Scaffold structure for the stdlib-docs are up: https://github.com/awvwgk/stdlib-docs

Nothing has been written yet and the color scheme has been stolen from fpm. Before we can consider this ready to be maintained under fortran-lang namespace we need some content, here are some suggestions:

- [ ] installation instruction in how-to (installing fypp, supported Fortran compilers)
- [ ] tutorials for a selected number of modules (at least two)
- [ ] workflow for contributing under design
- [ ] news announcement for v0.1.0
